### PR TITLE
🥳[Feature] Add Two Factor Authentication for Litecoin Card

### PR DESCRIPTION
**Pull request by kcw-grunt**
## Overview
The Litecoin Card API needs a higher degree of authentication than just login for Transfers. So, the Enable2FA Api needed to be added.

## Approach
There are a few structural changes tp make this happen

- The LitecoinCard API needed to be updated
- The Litewallet code needs to leverage that update (v0.8.0)
- A new Login UI needed to be added for the user to choose 2FA usage
- A new Enable UI Modal for the user to add the 2FA Code
- Add tests

## Screenshots/GIFs

|Before|After|
|-------|-----|
|Login|2FA-Login|

### CODE REVIEW

File: loafwalletTests/View Extension Tests/ViewExtensionTests.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
+//
+//  ViewExtensionTests.swift
+//  loafwalletTests
+//
+//  Created by Kerry Washington on 8/15/21.
+//  Copyright © 2021 Litecoin Foundation. All rights reserved.
+//
+
```
Comment (JCHeist) [inline loafwalletTests/View Extension Tests/ViewExtensionTests.swift]: Is this an extra file or was there some things we wanted to write in here?

File: loafwalletTests/Two Factor Auth Tests/Enter2FACodeViewModelTests.swift [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+    override func setUp() {
+        super.setUp()
+        viewModel = Enter2FACodeViewModel()
+    }
+
+    override func tearDownWithError() throws {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    }
+
+    func testExample() throws {
```
Comment (JCHeist) [inline loafwalletTests/Two Factor Auth Tests/Enter2FACodeViewModelTests.swift]: remove dead funcs (testExampe() and testPerformanceExample())

**Review by JCHeist — CHANGES_REQUESTED** [review]
Couple of testing files that need looked at to get some dead code reviewed

File: loafwalletTests/View Extension Tests/ViewExtensionTests.swift (hunk 1 above)
Comment (kcw-grunt) [reply loafwalletTests/View Extension Tests/ViewExtensionTests.swift]: Good catch. I was in my coding fever dream and while I should add tests, I can take this out as I'm really behind on this. Thanks.

File: loafwalletTests/Two Factor Auth Tests/Enter2FACodeViewModelTests.swift (hunk 2 above)
Comment (kcw-grunt) [reply loafwalletTests/Two Factor Auth Tests/Enter2FACodeViewModelTests.swift]: I'll remove. Tx.

**Comment by kcw-grunt** [comment]
@JasperHeist thanks...let me made changes and push them up soon (in 24 - 36 hrs)

**Comment by kcw-grunt** [comment]
@JasperHeist rebased the other develop merged PRs in:
- `f7ae472ba78372d352f91eb49acf6fa46297869f`
- `ca14eff0929b395620414b67607f4d93a8609ff0`

**Review by JCHeist — APPROVED** [review]
Sweet looks good here

**Pull request merged**